src/RTT_Format_Reader/CellDefs.cc

```diff
@@ -84,75 +79,67 @@ void CellDefs::readEndKeyword(ifstream & meshfile)
  *        definitions.
  */
 void CellDefs::redefineCellDefs(
-    vector_vector_int const & cell_side_types,
-    std::vector< std::vector< std::vector< size_t > > >
-    const & cell_ordered_sides)
-{
-    Insist(cell_side_types.size()
-           == static_cast<size_t>(dims.get_ncell_defs()),
-	   "Error in supplied cell redefinition side types data.");
-    Insist(cell_ordered_sides.size()
-           == static_cast<size_t>(dims.get_ncell_defs()),
-	   "Error in supplied cell redefinition ordered side data.");
+    vector_vector_int const &cell_side_types,
+    std::vector<std::vector<std::vector<size_t>>> const &cell_ordered_sides) {
+  Insist(cell_side_types.size() == static_cast<size_t>(dims.get_ncell_defs()),
+         "Error in supplied cell redefinition side types data.");
+  Insist(cell_ordered_sides.size() ==
+             static_cast<size_t>(dims.get_ncell_defs()),
```

Comment:
The current rule set will align at the opening parenthesis if we had a list of arguments separated by commas.  In this case the argument is an operation that will be indented on the continuation line.  Think of it as

```cpp
cell_ordered_sides = 
    Foo.bar();
```

Without the indent for the continuation line, the above wouldn't look right.

```cpp
cell_ordered_sides = 
Foo.bar();
```